Classification/include/CGAL/Classification/Mesh_feature_generator.h

```diff
@@ -275,15 +266,8 @@ class Mesh_feature_generator
   /// @}
 
   /// \cond SKIP_IN_MANUAL
```

Comment:
You should also remove this empty Doxygen block. /// \cond SKIP_IN_MANUAL -- /// \endcond